assets/pulpcore-api.prep

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/execlineb -P
+
+foreground {
+  mkdir -p /var/lib/pulp/tmp
+}
+foreground {
+  openssl ecparam -genkey -name prime256v1 -noout -out /var/lib/pulp/tmp/private.pem
```

Comment:
Run this only when the file doesn't already exist.

Reply:
Not so easy if you need to learn execline first.